index.js

```diff
@@ -51,24 +51,22 @@ function find_shims_in_package(pkgJson, cur_path, shims, browser) {
 
     // http://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders
     Object.keys(replacements).forEach(function(key) {
+        var val;
         if (replacements[key] === false) {
-            return shims[key] = __dirname + '/empty.js';
+          val = __dirname + '/empty.js';
+        } else {
```

Comment:
My bad, updated the indent width. There aren't any other `else` statements in the code, so I indented it like your `catch` statements. Let me know if this isn't what you had in mind.